payload-command/render/config.go

```diff
@@ -26,5 +28,13 @@ func readFeatureGateV1OrDie(objBytes []byte) *configv1.FeatureGate {
 }
 
 func writeFeatureGateV1OrDie(obj *configv1.FeatureGate) string {
-	return runtime.EncodeOrDie(configCodecs.LegacyCodec(configv1.SchemeGroupVersion), obj)
+	asMap, err := runtime.DefaultUnstructuredConverter.ToUnstructured(obj)
+	if err != nil {
+		panic(err)
+	}
+	ret, err := json.MarshalIndent(asMap, "", "    ")
+	if err != nil {
+		panic(err)
+	}
+	return string(ret)
```

Comment:
Nit, this currently doesn't write a new line char at the end, how would you feel about
```suggestion
	return string(ret) + "\n"
```